### ISSUE TITLE
Error on load (init.lua:129)

### ISSUE THREAD

**Issue by benfrain** [mrjones2014/lighthaus.nvim#4]
Thanks for making this, went to try it out but get this error:

```
E5108: Error executing lua ...te/pack/paqs/start/lighthaus.nvim/lua/lighthaus/init.lua:129: attempt to index local 'options' (a ni
l value)
```

**Think** I have done everything accoding to readme correctly!

**Comment by mrjones2014** [comment]
Did you pass an options table to the `setup` function? I can fix this regardless and make it so you don't need to pass an options table if you want to use the defaults.

**Comment by mrjones2014** [comment]
Resolved by 56a61d6
